app/spatio_helper.py

```diff
@@ -86,13 +86,33 @@ def hseg_to_coords(hseg):
     Returns:
         A list of dictionary of coordinates.
     """
-    unique_cords = []
+    cycleDict = dict()
```

Comment:
* variable names should be lowercase with underscores: cycleDict -> cycle_dict
* dict() is a built-in, takes longer to interpret, use `{}`